package.json

```diff
@@ -87,8 +87,7 @@
     "react-dom": "16.10.0",
     "react-emotion": "9.2.12",
     "react-hot-loader": "4.12.11",
-    "react-router": "5.0.1",
-    "react-router-dom": "5.0.1",
+    "react-router-dom": "5.1.2",
```

Comment:
The best is always update dependencies (even if are minor) in separated PR. This helps to identify whether an upgrade has broke something. I learnt the lesson from the past.

Reply:
I see...next time then?